use reqwest::Error;

#[tokio::main]
async fn main() -> Result<(), Error> {
    println!("hello");
    Ok(())
}
